lib/quoil/repeat.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Quoil.Repeat do
+  def requested? ({ip_pinged, switches, log_file_name}) do
+    {nr,switches} = Map.pop(switches, :repeat, nil)
+    if nr != nil && nr > 0 do
+      repeat_pings({nr, switches[:wait], {ip_pinged, switches, log_file_name}})
+    end
+  end
+  
+
+  def repeat_pings({nr, rpt_interval, args}) do
+    Stream.interval(rpt_interval * 60 * 1000)
+    |> Stream.take(nr)
+    |> Enum.map(fn _ -> Quoil.CLI.run_ping(args)|>Quoil.ParseResults.parse_result|>Quoil.LogResults.write_log end)
```

Comment:
We're repeating the main app logic here.

I'd look at extracting the `repeat` and `wait` switches in `Quoil.CLI.main`, and doing this logic there.